internal/gcs/gcs.go

```diff
@@ -15,8 +16,19 @@ import (
 // colorizer
 var style aurora.Aurora
 
+func IsRunningInGCP() bool {
+	resp, err := http.Get("http://metadata.google.internal")
+	if err != nil {
+		return false
+	}
+	defer resp.Body.Close()
```

Comment:
there's no need to defer this call here...

Reply:
changed, thanks :)